community-css-themes.json

```diff
@@ -1346,5 +1346,13 @@
         "repo": "isax785/obsidian-sparkling-night",
         "screenshot": "img/sparkling_night.png",
         "modes": ["dark", "light"]
+    },
+     {
+        "name": "Olivier’s Theme",
+        "author": "Olivier Spinnler",
+        "repo": "OlivierPS/Olivier-s-Theme",
+        "screenshot": "main-screenshot.png",
+        "modes": ["dark", "light"],
+        "branch": "master"
```

Comment:
remove this `branch` parameter, it's no longer used.

Reply:
Done !